Makefile

```
@@ -63,6 +63,10 @@ help:
 	@echo   make gofix-changed      Apply Go fixes to changed packages
 	@echo   make gofix-check-changed Check Go fix drift on changed packages
 
+# Local binaries must run on the development host despite cross-compilation defaults.
+build backend e2e-build: export GOOS := $(shell go env GOHOSTOS)
```

checkmake (0.3.2)

[warning] Target "build backend e2e-build" should be declared PHONY.

(phonydeclared)